File: app/src/main/java/org/michaelbel/tjgram/data/entity/AttachImageResponse.java
package org.michaelbel.tjgram.data.entity;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.jetbrains.annotations.NotNull;

import java.io.Serializable;

public class AttachImageResponse implements Serializable {
    @Expose @SerializedName("type") public String type;
    @Expose @SerializedName("data") public AttachImage data;

    @NotNull
    @Override
    public String toString() {
        return "AttachImageResponse{" +
                "type='" + type + '\'' +
                ", data=" + data +
                '}';
    }
}

File: app/src/main/java/org/michaelbel/tjgram/data/entities/AttachImageResponse.kt
package org.michaelbel.tjgram.data.entities

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.io.Serializable

data class AttachImageResponse (
    @Expose @SerializedName("type") var type: String,
    @Expose @SerializedName("data") var data: AttachImage
): Serializable